assert-tables/assert_tables.py

```diff
@@ -52,6 +52,7 @@ def main(table_names: str, host: str, max_retries: int):
         try:
             #session.open_table(table_name)
             #Temporary workaround: This script is sufficient to check that the table exists
+            print(f"{table_name}={table_name}")
```

Comment:
This seems like a very strange logging message for a user.  I suggest removing or making a clear message.

Reply:
That was an accidental commit from some unrelated testing I was doing. I removed it with commit `513392f` before